VideoApi/VideoApi.Client/VideoApiClient.cs

```diff
@@ -748,7 +748,7 @@ public VideoApiClient(System.Net.Http.HttpClient httpClient)
     
         private Newtonsoft.Json.JsonSerializerSettings CreateSerializerSettings()
         {
-            var settings = new Newtonsoft.Json.JsonSerializerSettings();
+            var settings = ConfigureVhJsonSettings(new Newtonsoft.Json.JsonSerializerSettings {  });
```

Comment:
should this be `()` ?

Reply:
This is autogenerated code, beyond my control unfortunately